JitsiConference.js

```diff
@@ -1705,20 +1643,6 @@ JitsiConference.prototype.setLastN = function(lastN) {
     }
 };
 
-/**
- * Checks if the participant given by participantId is currently included in
- * the last N.
- * @param {string} participantId the identifier of the participant we would
- * like to check.
- * @return {boolean} true if the participant with id is in the last N set or
- * if there's no last N set, false otherwise.
- * @deprecated this method should never be used to figure out the UI, but
- * {@link ParticipantConnectionStatus} should be used instead.
- */
-JitsiConference.prototype.isInLastN = function(participantId) {
```

Comment:
Why is this function removed? Can we provide a reasonable replacement?

Reply:
Jvb doesn't send the `LastNEndpointsChangeEvent` message anymore when the endpoint signals support for source-names. The equivalent function for the new signaling format is [getForwardedSources](https://github.com/jitsi/lib-jitsi-meet/blob/master/JitsiConference.js#L1670).

Reply:
👍